linux_os/guide/services/xwindows/disabling_xwindows/xwindows_remove_packages/oval/shared.xml

```diff
@@ -2,8 +2,10 @@
   <definition class="compliance" id="xwindows_remove_packages" version="1">
     {{{ oval_metadata("Ensure that the default runlevel target is set to multi-user.target.") }}}
     <criteria>
+      {{%- if init_system != "systemd" or target_oval_version != [5, 10] %}}
```

Comment:
Shouldn't this be `init_system == "systemd" and target_oval_version != [5, 10]`?

Reply:
maybe yes, I wasn't sure. I just applied De Morgan's laws from this macro: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/xwindows/disabling_xwindows/xwindows_runlevel_target/oval/shared.xml#L0-L1

Reply:
Done

Reply:
Hmm, it seems to me that the macro is wrong, as a product with `init_system != systemd` would always end up including the `symlink_test` for the `/etc/systemd/...` path.

At the moment all products that define `init_system` use `systemd`, so not a big issue.